allennlp/data/tokenizers/character_tokenizer.py

```diff
@@ -46,20 +47,28 @@ def __init__(self,
         self._end_tokens = end_tokens or []
 
     @overrides
-    def tokenize(self, text: str) -> Tuple[List[str], List[Tuple[int, int]]]:
+    def tokenize(self, text: str) -> List[Token]:
         if self._lowercase_characters:
             text = text.lower()
         if self._byte_encoding is not None:
             # We add 1 here so that we can still use 0 for masking, no matter what bytes we get out
             # of this.
-            tokens = [c + 1 for c in text.encode(self._byte_encoding)]
+            tokens = [Token(text_id=c + 1) for c in text.encode(self._byte_encoding)]
         else:
-            tokens = list(text)  # type: ignore
+            tokens = list(map(Token, list(text)))
```

Comment:
same comment as before, except even moreso here, as the contrast with the list comprehension two lines above is jarring

Reply:
Done.